packages/integrations/README.md

```diff
@@ -2,7 +2,65 @@
 
 Configu SDK for Node.js published to [npm](https://www.npmjs.com/package/@configu/node).
 
-## Install
+
+# Integrations Package
+
+## What is an Integration?
+
+An integration in Configu is a module that allows Configu to interact with external systems, services, or tools. Integrations enable seamless management and collaboration on software configuration data across various environments and systems. They provide a standardized way to connect Configu with other platforms, ensuring consistent and efficient configuration operations.
+
+## How to Use an Integration in the New CLI Syntax
+
+Using an integration in the new CLI syntax is straightforward. Here is an example of how to use an integration:
+
+```sh
+configu integration <integration-name> <command> [options]
+```
+
+For example, to use the AWS SSM integration to fetch configuration data, you can use the following command:
+
+```sh
+configu integration aws-ssm fetch --path /my/config/path
+```
+
+## How to Create a New Integration Package
+
+Creating a new integration package involves the following steps:
+
+1. **Set Up the Project**: Create a new directory for your integration package and set up the necessary files.
+2. **Define the Integration**: Implement the integration logic by creating the necessary classes and methods.
+3. **Register the Integration**: Register your integration with Configu so that it can be used via the CLI.
+4. **Test the Integration**: Write tests to ensure that your integration works as expected.
+5. **Document the Integration**: Provide documentation on how to use your integration.
+
+Here is an example structure for a new integration package:
+
+```
+my-integration/
+├── __init__.py
+├── integration.py
+├── cli.py
+├── tests/
+│   └── test_integration.py
+└── README.md
+```
+
+## Where Do Those Integrations Deploy?
+
+Integrations are deployed as part of the Configu ecosystem. They can be installed and used via the Configu CLI. When you create a new integration, you can publish it to the Configu registry or any other package registry that Configu supports. This allows other users to easily install and use your integration.
+
+To deploy an integration, you can use the following command:
+
+```sh
+configu integration deploy <integration-name>
```

Comment:
there's no deploy command.
this repository has a pipeline that publishes all integrations as a public release.
our new CLI can retrieve those integrations on-demand.

any developer can also develop his own custom integration on his own private repository / organization, if they have very specific and unique integration that is only relevant to their company.
but - we encourage developers to contribute integration here and to share them among any other configu users.